src/modules/AutoinstClone.rb

```diff
@@ -50,32 +51,22 @@ def multipath_in_use?
     end
 
     # General options
+    #
     # @return [Hash] general options
     def General
       Yast.import "Mode"
       Mode.SetMode("normal")
 
       general = {}
-
       general["mode"] = { "confirm" => false }
-
-      general["signature-handling"] = {
-        "accept_unsigned_file"         => true,
-        "accept_file_without_checksum" => true,
-        "accept_unknown_gpg_key"       => true,
-        "accept_verification_failed"   => false,
-        "import_gpg_key"               => true,
-        "accept_non_trusted_gpg_key"   => true
-      }
-      general["storage"] = {
-        "start_multipath" => multipath_in_use?
-      }
+      general["storage"] = { "start_multipath" => true } if multipath_in_use?
```

Comment:
what is default? if we can recognize usually properly multipath, I think we should keep it empty and let yast detect it itself, not?

Reply:
You explicitly need to start multipath (using the profile or Linuxrc). In a regular installation, YaST will ask you about enabling multipath (if detected). But with AutoYaST, it simply does not use multipath unless you tell it to do it. See [the activation callback](https://github.com/yast/yast-autoinstallation/blob/bsc-1171343-set-signature-handling/src/lib/autoinstall/activate_callbacks.rb#L34).

The idea of this change to clone the multipath setting only when it is relevant (when it is `true`). Otherwise, it is just adding noise to the profile.